regression-test/src/app/view/page.tsx

```diff
@@ -24,7 +24,7 @@
 
 'use client'
 import React from 'react'
-import { View as vw } from 'instructure-ui/ui-view/es/index'
```

Comment:
these and (others in the regression tests) are intentionally imported from the `/es` folder, so we test the compiled code